### PR TITLE
AccessControl: Add displayname field to Role

### PR DESCRIPTION
**Pull request by eleijonmarck**
**What this PR does / why we need it**:
To be able to make a more human readable view of the role

Which issue(s) this PR fixes:
N/A

### CODE REVIEW

File: pkg/services/accesscontrol/models.go [hunk 1]
```diff
@@ -59,8 +63,35 @@ func (r Role) Global() bool {
...
+func (r Role) GetFallbackDisplayName() string {
+	return fallbackDisplayNameHeuristic(r.Name)
+}
+
+// heuristic for fallback display name
+func fallbackDisplayNameHeuristic(rName string) string {
+	// removing prefix for fixed roles
+	rNameWithoutPrefix := strings.Replace(rName, FixedRolePrefix, "", 1)
+	return strings.TrimSpace(strings.Replace(rNameWithoutPrefix, ":", " ", -1))
+}
```
Comment (eleijonmarck) [inline pkg/services/accesscontrol/models.go:87]: displayName heuristic for fallback behaviour

File: pkg/services/accesscontrol/models.go [hunk 2]
```diff
@@ -59,8 +63,35 @@ func (r Role) Global() bool {
 	return r.OrgID == GlobalOrgID
 }
 
+func (r RoleDTO) IsFixed() bool {
+	return strings.HasPrefix(r.Name, FixedRolePrefix)
+}
```
Comment (kalleep) [inline pkg/services/accesscontrol/models.go:68]: Nit: Place methods for a struct under corresponding type

File: pkg/services/accesscontrol/models.go [hunk 3]
```diff
@@ -59,8 +63,35 @@ func (r Role) Global() bool {
...
+
+func (r Role) IsFixed() bool {
+	return strings.HasPrefix(r.Name, FixedRolePrefix)
+}
+
+func (r RoleDTO) GetFallbackDisplayName() string {
+	return fallbackDisplayName(r.Name)
+}
```
Comment (kalleep) [inline pkg/services/accesscontrol/models.go:76]:
```suggestion
func (r RoleDTO) GetDisplayName() string {
   if r.IsFixed() && r.DisplayName == "" {
		return fallbackDisplayName(r.Name)
	}
	return r.DisplayName
}
```

**Review by kalleep — APPROVED** [review]
LGTM!

**Pull request merged**